plugins/plugin-docker/che-plugin-docker-client/src/main/java/org/eclipse/che/plugin/docker/client/DockerConnector.java

```diff
@@ -168,6 +171,41 @@ public Version getVersion() throws IOException {
     }
 
     /**
+     * Method returns list docker containers which wos filtered by query parameters from {@link ListContainersParams}
+     *
+     * @throws IOException
+     *         in case error parsing response from docker api
+     */
+    public ContainerFromList[] listContainers(ListContainersParams params) throws IOException {
```

Comment:
Please add method with no argument because it will be used mostly. So simplified code of its execution will be appreciated.